modules/eop-monitoring/main.tf

```diff
@@ -0,0 +1,237 @@
+locals {
+  manager_service_name    = "eop-manager"
+  tileserver_service_name = "eop-tileserver"
+
+  manager_log_group_name    = "/${var.account_name}/ecs/${local.manager_service_name}"
+  tileserver_log_group_name = "/${var.account_name}/ecs/${local.tileserver_service_name}"
+}
```

Comment:
This felt a bit painful, trying to decide what should be bought in as dependencies from the other modules vs calculated here. 

I pulled in what I could, but there were a few things not exposed from the Gruntworks modules